src/scripts/main.js

```diff
@@ -1,3 +1,27 @@
 'use strict';
 
-// write your code here
+const staffTable = document.querySelector('table');
+
+cloneTablbeColumn(staffTable, 1, 4);
+
+function cloneTablbeColumn(table, originIndex, insertBeforeIndex) {
+  for (let i = 0; i < table.rows.length; i++) {
+    const row = table.rows[i];
+    const rowItems = row.children;
+    const rowItem = rowItems[originIndex];
+
+    if (rowItem.tagName === 'TH') {
+      const th = document.createElement('th');
+
+      th.textContent = rowItem.textContent;
+      row.insertBefore(th, rowItems[insertBeforeIndex]);
+    }
+
+    if (rowItem.tagName === 'TD') {
+      const td = document.createElement('td');
+
+      td.textContent = rowItem.textContent;
+      row.insertBefore(td, rowItems[insertBeforeIndex]);
+    }
```

Comment:
create some function that will accept element name, text and item where insert
like

```
const insert = (tagName, text, rowToInsert, insertBefore) => {
  const element = document.createElement(tagName.toLowerCase());

  element.textContent = text;
  rowToInsert.insertBefore(td, insertBefore);
}
```

and use it like

```suggestion
  insert(rowItem.tagName, rowItem.textContent, row, rowItems[insertBeforeIndex])
```

Reply:
Ok, done